AUTHORS

```diff
@@ -634,6 +637,7 @@ Eng Zer Jun <engzerjun@gmail.com>
 Enguerran <engcolson@gmail.com>
 Eohyung Lee <liquidnuker@gmail.com>
 epeterso <epeterson@breakpoint-labs.com>
+er0k <er0k@er0k.net>
```

Comment:
Looks like some "not real name" commits sneaked in. For some of those we can add a `.mailmap` entry (but looks like we don't have the correct information for all of them)